src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsType.php

```diff
@@ -79,35 +79,74 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('enable_invoices', SwitchType::class)
-            ->add('enable_tax_breakdown', SwitchType::class)
-            ->add('enable_product_images', SwitchType::class)
+            ->add('enable_invoices', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Enable invoices', 'Admin.Orderscustomers.Feature'),
+                'help' => $this->trans('If enabled, your customers will receive an invoice for the purchase.', 'Admin.Orderscustomers.Help'),
+            ])
+            ->add('enable_tax_breakdown', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Enable tax breakdown', 'Admin.Orderscustomers.Feature'),
+                'help' => $this->trans('If required, show the total amount per rate of the corresponding tax.', 'Admin.Orderscustomers.Help'),
+            ])
+            ->add('enable_product_images', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Enable product image', 'Admin.Orderscustomers.Feature'),
+                'help' => $this->trans('Adds an image in front of the product name on the invoice.', 'Admin.Orderscustomers.Help'),
```

Comment:
@Junebyun, some help texts are written in the third person singular while others are not. It might be nice to have it harmonized. :-)

Reply:
```suggestion
                'help' => $this->trans('Add an image in front of the product name on the invoice.', 'Admin.Orderscustomers.Help'),
```